src/SaaS.SDK.Client.DataAccess/Contracts/IMeteredDimensionsRepository.cs

```diff
@@ -17,5 +17,7 @@ public interface IMeteredDimensionsRepository : IDisposable, IBaseRepository<Met
         /// <param name="planId">The plan identifier.</param>
         /// <returns></returns>
         List<MeteredDimensions> GetDimensionsFromPlanId(string planId);
+        List<MeteredDimensions> GetDimensionsByPlanId(int planId);
```

Comment:
We can have the method name as GetDimensionsByPlan ( string planId) as well as GetDimensionsByPlan ( int planId) . Method overloading.